user-service/Dockerfile

```
@@ -0,0 +1,45 @@
+
+COPY common/src common/src
+COPY common-jpa/src common-jpa/src
+COPY common-event/src common-event/src
+COPY common-security/src common-security/src
+COPY user-service/src user-service/src
+
+RUN --mount=type=cache,target=/root/.gradle \
+    ./gradlew :user-service:bootJar --no-daemon -x test
+
+FROM eclipse-temurin:21-jre-alpine
+RUN apk add --no-cache curl
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)